include/RAJA/policy/openmp_target/reduce.hpp

```diff
@@ -193,6 +193,9 @@ struct TargetReduce
   {
   }
 
+#ifdef __ibmxl__ // implicit declare target doesn't pick this up
```

Comment:
We don't have a precedent for this, but perhaps this comment would be good with a `TODO:` prefix?

```suggestion
#ifdef __ibmxl__ // TODO: implicit declare target doesn't pick this up
```